ui/token/TokenNftMarketplaces.tsx

```diff
@@ -1,16 +1,23 @@
 import { Image, Link, Skeleton, Tooltip } from '@chakra-ui/react';
 import React from 'react';
 
+import type { AddressMetadataTagFormatted } from 'types/client/addressMetadata';
+
 import config from 'configs/app';
+import ActionButton from 'ui/shared/ActionButton/ActionButton';
 import DetailsInfoItem from 'ui/shared/DetailsInfoItem';
+import TextSeparator from 'ui/shared/TextSeparator';
 
 interface Props {
   hash: string | undefined;
   id?: string;
   isLoading?: boolean;
+  actionData?: AddressMetadataTagFormatted['meta'];
+  source: 'NFT collection' | 'NFT item';
```

Comment:
Strictly speaking, this property is only required for experiments. Therefore, shouldn't it be optional?

Reply:
The experiment is temporary and with high probability will be used permanently later, and the component is used only where it is necessary to pass this parameter, so I don't think it should be made optional